README.md

```diff
@@ -12,17 +12,41 @@ Enjoy!
 - Ctrl + Shift + t
 - Cmd + Shift + t (Mac)
 
-## Redine shortcuts:
+## Change keybinding:
 
+Within VSCode (last tested in v1.56.1):
+1. Open _Keyboard Shortcuts_ menu (`ctrl+k ctrl+s` for windows, `cmd+k cmd+s` for mac)
+1. Search for `Go to test`
+1. Hover over this extension's shortcut (identified by `extension.goToTest`) and click on the edit icon to the left (looks like a pencil)
+1. Press a key combination and save to overwrite the default keybinding
```

Comment:
Shouldnt the numbers be `1. ... , 2. ..., 3. ....`?

Reply:
Ah, yes. I've made it a habit to use only `1.` to make it easier to re-order the list, GH renders correctly anyway. But you're right. Using incremental numbers is better. Fixed in https://github.com/SimonBo/rails-go-to-test/pull/4/commits/587a061a11c02856562714af6307d75a06b4e74a